src/pkg/util/fs/layout/layer/underlay/underlay.go

```diff
@@ -136,7 +136,8 @@ func (u *Underlay) createLayer(rootFsPath string, system *mount.System) error {
 	return u.session.Update()
 }
 
-func (u *Underlay) duplicateDir(dir string, system *mount.System) error {
+func (u *Underlay) duplicateDir(dir string, system *mount.System, existingPath string) error {
```

Comment:
If I understand correctly, there is no need to add an argument here, instead of doing `if binds > 50 && existingPath != ""` you could have `if binds > 50 && dir != "/"`

Reply:
It's true you could check dir != "/" but the more important reason for the argument is to include it in the warning message.

Reply:
And `dir` is not enough for the warning message ?

Reply:
I don't think so because it helps people to know which bind caused the duplication, not just the name of the directory that was duplicated.